spec/lib/web_of_science/process_records_spec.rb

```diff
@@ -74,10 +74,10 @@
       expect { processor.execute }.to change { Contribution.count }
     end
     it 'creates new contribution in the pub_hash[:authorship]' do
-      uids = processor.execute
-      uids.each do |uid|
-        pub = Publication.for_uid(uid)
-        expect(pub.pub_hash).to include(:authorship)
+      processor.execute
```

Comment:
why not just do `processor.execute.each` ?

Reply:
Ah, I see, you are just using the `uid`s from the fixture files.

Reply:
This uses the original set of records, whereas the return from the processing could be a filtered set of results.  In this spec, all of the records should create new pubs, so it's got to check that they all exist etc.